presto-native-execution/presto_cpp/presto_protocol/core/presto_protocol_core.yml

```diff
@@ -314,3 +314,5 @@ JavaClasses:
   - presto-main/src/main/java/com/facebook/presto/connector/system/SystemTransactionHandle.java
   - presto-spi/src/main/java/com/facebook/presto/spi/function/AggregationFunctionMetadata.java
   - presto-function-namespace-managers/src/main/java/com/facebook/presto/functionNamespace/json/JsonBasedUdfFunctionMetadata.java
+  - presto-native-sidecar-plugin/src/main/java/com/facebook/presto/sidecar/nativechecker/PlanConversionFailureInfo.java
+  - presto-native-sidecar-plugin/src/main/java/com/facebook/presto/sidecar/nativechecker/PlanConversionResponse.java
```

Comment:
@aditi-pandit can you confirm these is the right place to define messages for the sidecar plugin?

Reply:
@BryanCutler : This is fine. 

If you prefer a separate yml file for the native side-car classes that is okay as well. Though it seems like there aren't many such native-side-car classes right now, so we can add them to the core yml file for a start.

Reply:
Let's leave it here if that's ok. There isn't anything else now, so we can separate later if more is added.